tools/nightly.py

```diff
@@ -75,7 +86,7 @@
     Version = None  # type: ignore[assignment,misc]
 
 
-REPO_ROOT = Path(__file__).absolute().parent.parent
+REPO_ROOT = Path.cwd().absolute()
```

Comment:
The script may not always be run from the repository root. Because the wheel so files will be copied to the local dir, and a `path.pth` will be put in site-packages. We should ensure `REPO_ROOT` is the true path to the local git repo.